### ISSUE TITLE
Models onf Huggingface

### ISSUE THREAD

**Issue by daniel-bogdoll** [xiuqhou/Relation-DETR#25]
### Question

Hey,

are you planning to release the models of Salience-DETR and Relation-DETR also on Huggingface? I see that you have already published the dataset there :) Would be great for easy usage!

https://huggingface.co/xiuqhou

**Comment by xiuqhou** [comment]
Yes, I'd like to make Salience-DETR and Relation-DETR available on Huggingface. But I'm a little busy recently, once I'm done, I will sort out the code and release it on Huggingface as soon as possible.
